install/kubernetes/helm/istio-remote/charts/sidecarInjectorWebhook/Chart.yaml

```diff
@@ -0,0 +1,11 @@
+name: sidecarInjectorWebhook
+version: 0.8.0
+tillerVersion: ">=2.7.2"
+description: Helm chart for sidecar injector webhook deployment
+keywords:
+  - istio
```

Comment:
Please add `security` as well.

Reply:
@gyliu513 I prefer not to carry deltas in the istio-remote chart.  The update model should be a simple copy and paste.  that said, if you think security belongs as a keyword, might submit that to the main istio helm chart.

Reply:
It is a straight copy/paste. I'd personally prefer to use `helm dependency` but that's not a pattern that is currently in use for the project. I could change it to symlinks if we think t's a better approach to avoid drift but I generally don't like them in git repos.

Reply:
@sdake @gyliu513 --what do you guys think?  IMO, relative path symlinks are ok and easier to review--explicitly ensuring "there's no change here".

Reply:
+1 to symlinks.